lib/index.ts

```diff
@@ -45,37 +45,39 @@ export function splitBuildStream(
 			next: () => void,
 		): void => {
 			// Find the build context that this file should belong to
-			const matchingTask = _.find(tasks, (task) => {
+			const matchingTasks = _.filter(tasks, (task) => {
 				if (task.external) {
 					return false;
 				}
 				return PathUtils.contains(task.context!, header.name);
 			});
 
-			if (matchingTask != null) {
+			if (matchingTasks.length === 0) {
```

Comment:
I would check the happy-path first (i.e. `matchingTasks.length > 0`) like it was before to reduce the diff somewhat.

Reply:
Yeah good idea.